app/GraphQL/Directives/MorphToRelationDirective.php

```diff
@@ -57,7 +57,7 @@
 use LastDragon_ru\LaraASP\GraphQL\SearchBy\Exceptions\OperatorInvalidArgumentValue;
 use LastDragon_ru\LaraASP\GraphQL\SearchBy\Definitions\SearchByOperatorPropertyDirective;
 
-class MorphToRelationDirective extends BaseOperator
+final class MorphToRelationDirective extends BaseOperator
```

Comment:
What is the purpose of making this final?

Reply:
I made it final because the code the package generated kept being marked final so I changed it to make it consistent, but I don't really like final anyway so I'm just going to remove it from all of the directives and scalars.